pkg/storaged/partitions/partition-table.jsx

```diff
@@ -109,3 +112,13 @@ const PartitionsCard = ({ card }) => {
         </StorageCard>
     );
 };
+
+const UnexpectedPartitionsCard = ({ card }) => {
+    return (
+        <StorageCard card={card}>
+            <CardBody>
+                {_("Partitions are not supported on this block device. If it is used as a disk for a virtual machine, the partitions must be managed by the operating system inside the virtual machine.")}
```

Comment:
```suggestion
                <Alert isInline isPlain variant="info" title={_("Unexpected partitions")}>
                    <p>{_("Partitions are not supported on this block device. If it is used as a disk for a virtual machine, the partitions must be managed by the operating system inside the virtual machine.")}</p>
                </Alert>
```

![image](https://github.com/user-attachments/assets/b5edc9cc-32a8-4895-a995-1bf6650d0d65)

Reply:
Excellent, thanks!